### ISSUE TITLE
question about the result

### ISSUE THREAD

**Issue by 1245244103** [allegro/allRank#30]
I standized the features and disposed the config.json as follow:
![QQ图片20210415144124](https://user-images.githubusercontent.com/58716304/114825342-af187d00-9df8-11eb-9f29-a275e9211dd8.png)
I run on the FOLD 1 of  MSLR-WEB30K and just get  0.502 of ndcg@5 on the test set. Is there any  procedure that i miss？
Thanks a lot！

**Comment by PrzemekPobrotyn** [comment]
we are currently working on a reproducibility guide and will post it here on github soon, hopefully it will answer your question

**Comment by 1245244103** [comment]
> we are currently working on a reproducibility guide and will post it here on github soon, hopefully it will answer your question

It is a good news. I am willing to wait for it!

**Comment by sadaharu-inugami** [comment]
Did you use the latest 1.4.1 version to produce these results? In this version, we changed the way we calculated NDCG. Following LightGBM and XGBoost's default behaviour (AFAIK, this was the way the results were calculated e.g. in the LambdaLoss paper), we now set NDCG = 1.0 for lists with no relevant items, where IDCG == 0.0.

**Comment by sadaharu-inugami** [comment]
Also, I can confirm that we are working on the reproducibility guide and will be posting it very soon - maybe as soon as today or tomorrow.
